packages/core-generator/src/generator.ts

```diff
@@ -105,6 +103,8 @@ import {
 } from './types';
 import { getExpression } from './utils/expressions';
 import { getModuleRelativePath, resolveModule } from './utils/path-utils';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const format = require('prettier-eslint');
```

Comment:
Why require is used here instead of import?

Reply:
The mocha uses ts-node to compile ts into js. The prettier-eslint package doesn't provide types info. In this case, the TS7016 error is raised. I try to add file with declare module but it doesn't work as proposed in the following link
https://stackoverflow.com/questions/51610583/ts-node-ignores-d-ts-files-while-tsc-successfully-compiles-the-project

So the using require without import is the only one workaround.